### PR TITLE
Update layout wallet main page, update logic segmented control and logic scroll loading

### LINKED ISSUE #356: Updated some layouts for mobile views
1. need center
image

2. can't swipe on bottom page
image

3. center back button
image

4. loading not stop 
image

5.  need unfocus editor when click outside for hide keyboard
image

### PR DESCRIPTION
**Pull request by JustinBeBoy**
Resolve #356

### CODE REVIEW

File: ui/cryptomaterial/segmented_control.go [hunk 1]
```diff
@@ -133,6 +133,21 @@ func (sc *SegmentedControl) Layout(gtx C, body func(gtx C) D, isMobileView ...bo
...
 
+func (sc *SegmentedControl) layoutContent(body layout.Widget) layout.Widget {
+	return func(gtx C) D {
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				dims := body(gtx)
+				return dims
+			}),
+			layout.Flexed(1, func(gtx layout.Context) layout.Dimensions {
+				// 600px use to fill height for swip action
```
Comment (dreacot) [inline ui/cryptomaterial/segmented_control.go:144]:
```suggestion
				// 600px use to fill height for swipe action
```

**Comment by dreacot** [comment]
> center back button

the previous position was as per the figma design

**Comment by JustinBeBoy** [comment]
> > center back button
> 
> the previous position was as per the figma design

I know, but it is not friendly with mobile version, it looks like the UI bug

File: ui/cryptomaterial/segmented_control.go (hunk 1 above)
Comment (dreacot) [reply ui/cryptomaterial/segmented_control.go:144]: @JustinBeBoy _

**Review by dreacot — APPROVED** [review]

**Pull request merged**